frontend/src/components/register/index.js

```diff
@@ -1,109 +1,63 @@
-import React, { useRef, useState } from 'react'
-import { Controller, useForm } from 'react-hook-form'
+import React, { useState } from 'react'
+import { useForm } from 'react-hook-form'
 import axios from 'axios'
 
 function Index () {
+  const [emailError, setEmailError] = useState(false)
   const { register, handleSubmit, errors, getValues } = useForm()
-  const password = useRef({})
-  const [emailValid, setEmailValid] = useState()
-  const [passwordValid, setPasswordValid] = useState()
-  const [confirmValid, setConfirmValid] = useState()
-  const [passwordConfirm, setPasswordConfirm] = useState(false)
-  /*
-  ref={register({
-      validate: {
-        emailEqual: value => (value === getValues().email) || 'Email confirmation error!',
-      }
-    })}
-    */
+  const onSubmit = async (data) => {
+    axios.post("/register", {
+      name: data.fname,
+      email: data.email,
+      password: data.password,
+    })
+    .then(function (response){
+      if (response.data.error) setEmailError(true)
+    })
+    .catch(function (err) {
+      console.log(err)
```

Comment:
Console.logging the error message here is fine but normally you want to have some sort of error message displayed to the user. Typically you want console.logs only for errors like here.